### PR TITLE
Hospital Beds Face to Face Questionnaire

### PR DESCRIPTION
**Pull request by yuliumitre**
This PR add face to face encounter questionnaire form for the Hospital Bed ordering. It also uses the valueset defined from vsac.

### CODE REVIEW

File: HospitalBeds/R4/resources/Quesionnaire-R4-HospitalBedsFaceToFace.json [hunk 1]
```diff
@@ -0,0 +1,1729 @@
...
+        }
+      ]
+    },
+    {
+      "linkId": "12",
+      "type": "text",
+      "text": "Treatment plan"
+    },
+    {
+      "linkId": "13",
```
Comment (yunwwang) [inline HospitalBeds/R4/resources/Quesionnaire-R4-HospitalBedsFaceToFace.json:1573]: Do we need to get details for each items here? Template and CDE ask for text only.

File: HospitalBeds/R4/resources/Quesionnaire-R4-HospitalBedsFaceToFace.json [hunk 2]
```diff
@@ -0,0 +1,1729 @@
...
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HospitalBedsPrepopulation\".FixedHeightBedsOrdered"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "11.2",
```
Comment (yunwwang) [inline HospitalBeds/R4/resources/Quesionnaire-R4-HospitalBedsFaceToFace.json]: May be put this one as sub item of 11.1.

**Comment by yuliumitre** [comment]
@yunwwang Is there anything else? Please let me know. Thanks.

**Review by lpdecelles — CHANGES_REQUESTED** [review]
image

When I click the F2F link in the CDS Hooks card I see the error above.

**Comment by lpdecelles** [comment]
After I updated my value sets it seems to get further. I did have a couple questions. 
1) Should DTR fail if some of the value sets are not found?
2) It seems like some of the pre-population is not working. Am I missing something?

**Comment by yuliumitre** [comment]
Hi, Larry, I merged master to this branch as it has been a while. 

The form got displayed fine for me. Please see the attached image. Regarding the pre-populated fields, do you have any specific concerns?  Please let me know. Thanks. 

Screen Shot 2020-05-26 at 10 40 43 AM

**Comment by lpdecelles** [comment]
OK, looks good after a forward merge from master

**Pull request merged**